progsrc/android-template/build.gradle

```diff
@@ -13,6 +13,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url 'https://maven.google.com' }
```

Comment:
Couldn't find some dependencies when first runnig `gradlew` via command line. adding the google repository fixed the problem (found the solution on some stackoverflow post)